### ISSUE TITLE
Cannot resize Imposter Tab

### ISSUE THREAD

**Issue by Radivarig** [zhangjt93/godot-imposter#8]
Godot `4.3-stable`

As mentioned in #6 the tab cannot be resized properly, it takes up too much space and is hiding the right sidebar completely.
As if it has minimum width/height set somewhere, or hardcoded canvas nodes dimensions instead of fill to container or something.

![image](https://github.com/user-attachments/assets/0342f003-51e1-4ee2-95c0-39ebc921fed9)

**Comment by zhangjt93** [comment]
Yeah, I set the custom_minimum_size property in the scene to make sure all the content can be shown. For example: addons/Imposter/imposter.tscn.

**Comment by lucasthomaz97** [comment]
On my notebook it's also happening, i have to use this addon and start pressing tab until i leave this screen, since even the lower options bar from godot don't appear to me, i tried lowering the size of the container but it don't solve this problem to me.
I really liked the function, but the interface is not working well.

**Comment by zhangjt93** [comment]
When building the plugin, to make sure the GUI looks good, some nodes have a custom_minimum_size set. But this can make the size of the nodes fixed, which isn't really a coding issue. You can find and tweak the custom_minimum_size property of the nodes to fit your screen. Here are the files you'll need to check out:

addons/Imposter/imposter.gd
addons/Imposter/imposter.tscn
addons/Imposter/imposter/scenes/batch_bake.tscn
addons/Imposter/imposter/scenes/dilatate.tscn
addons/Imposter/imposter/scenes/single_bake.tscn

**Comment by monxa** [comment]
I'd consider a `ScrollContainer` in-between.

**Comment by Radivarig** [comment]
@zhangjt93 Apologies if I'm misunderstanding, but from your 2 replies it seems to me that you're expecting every user to "just tweak 5 files" based on their arbitrary screen/window size every time they import the addon or the sizes change.

I'm not saying you should necessarily be the one fixing it but to acknowledge that the issue should be resolved by this repository and that the UI should be responsive and work out of the box, otherwise this addon is unusable.

**Comment by lucasthomaz97** [comment]
Yes, it's kind of unusable if you don't tweak each scene.
Yesterday i did it, made each widget expand to fill the viewport size. I can make a pull request for you to test it and check it out.


https://github.com/user-attachments/assets/8daa734f-bad1-4d2e-93bd-218cc4b0f5c6

**Comment by lucasthomaz97** [comment]
Here's the pull request: https://github.com/zhangjt93/godot-imposter/pull/9

**Comment by zhangjt93** [comment]
> @zhangjt93 Apologies if I'm misunderstanding, but from your 2 replies it seems to me that you're expecting every user to "just tweak 5 files" based on their arbitrary screen/window size every time they import the addon or the sizes change.
> 
> I'm not saying you should necessarily be the one fixing it but to acknowledge that the issue should be resolved by this repository and that the UI should be responsive and work out of the box, otherwise this addon is unusable.

Yeah, as a plugin, I totally see where you're coming from. Unfortunately, my day job has been eating up a ton of my time lately, so I haven't had much time to tackle these issue. Luckily, @lucasthomaz97  put in some hard work, and it looks like the issue been sorted out. Big thanks to @lucasthomaz97  for stepping up! I'll do my best to find some time soon to make tests and merge in.

**Comment by zhangjt93** [comment]
I fixed this in my latest update—feel free to take a look.
